prometheus-exporters/vrops-exporter/templates/collector-configmap.yaml

```diff
@@ -22,6 +22,7 @@ data:
     #    - 'VMStatsDefaultCollector'
 
     ClusterPropertiesCollector:
+    # Prefix: vrops_cluster_
```

Comment:
why is it commented out?

Reply:
Just an info. Added `INFO` to the comment.